dsgrid/config/dataset_config.py

```diff
@@ -498,6 +493,10 @@ def load_dimensions(self, dimension_manager):
 
         """
         self._dimensions.update(dimension_manager.load_dimensions(self.model.dimension_references))
+        check_uniqueness((x.model.name for x in self._dimensions.values()), "dimension name")
+        check_uniqueness(
+            (x.model.query_name for x in self._dimensions.values()), "dimension query name"
+        )
```

Comment:
Can we exclude trivial dimensions from this uniqueness check? When we aggregate to "all sectors" or "all subsectors" I'd like the drop-down for resolution to say "None," and similarly you would never say "where all-sectors = X" or "where all-sectors IN (X)".